core/file.rbs

```diff
@@ -984,7 +984,7 @@ File::Constants::WRONLY: Integer
 class File::Stat < Object
   include Comparable
 
-  def initialize: (String file) -> Object
```

Comment:
As far as I can see from the existing RBS file, the return type of the `initialize` method is `void` in some places.

I believe that the return value of a constructor can only be an instance of the class itself.

Therefore, the return value of the constructor should be `self`.

What are the cases in which it is appropriate to use `void`?

Reply:
Thanks for your comment.
Now I know the meaning of `new`, `allocate` and `initialize` in Ruby.